README.md

```diff
@@ -49,6 +50,62 @@ Please see the [roadmap](./docs/roadmap.md).
 
 > Read more [documentation](./docs/index.md)
 
+## Usage
```

Comment:
I think a small bit of this new documentation ought to live here (maybe even most of what's been written out so far), but we have a docs folder which I think is more relevant when it starts to get in depth - e.g. when laying out the API